pkg/cri/sbserver/restart.go

```diff
@@ -116,7 +116,7 @@ func (c *criService) recover(ctx context.Context) error {
 
 		status, err := controller.Status(ctx, sbx.ID, false)
 		if err != nil {
-			log.G(ctx).WithError(err).Error("failed to recover sandbox state")
+			log.G(ctx).WithError(err).Errorf("failed to recover sandbox state for %q", sbx.ID)
```

Comment:
I would still prefer `WithField("sandbox", sbx.ID)` since it is easier to collect multiple logs and could also add it higher up to the context.

Reply:
ack

Reply:
Moved to a field, and adjusted two of the current logs that also included the container or sandbox ID as format strings.